hadoop-hdds/rocks-native/pom.xml

```
@@ -436,5 +449,24 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>noNativeTest</id>
```

[Conventionally](https://maven.apache.org/guides/introduction/introduction-to-profiles.html#profile-order) maven profile id uses hyphen with all lower cases:

```suggestion
            <id>disable-native-testing</id>
```

---

done